scripts/comfyui.py

```diff
@@ -0,0 +1,60 @@
+import gradio as gr
+
+import modules.scripts as scripts
+import sys
+from lib_comfyui import webui_callbacks, webui_settings, global_state
+from comfyui_custom_nodes import webui_postprocess_input, webui_postprocess_output
+from lib_comfyui.polling_client import ComfyuiNodeWidgetRequests
+from lib_comfyui.queue_tracker import PromptQueueTracker
+
+base_dir = scripts.basedir()
+sys.path.append(base_dir)

```

Comment:
Do we still need to sys.append this? I thought the webui fixed this a while ago... was it ever necessary to do this in fact?